src/CommunityToolkit.Maui.MediaElement.Analyzers/UseCommunityToolkitMediaElementInitializationAnalyzer.cs

```diff
@@ -0,0 +1,88 @@
+﻿using System;
+using System.Collections.Immutable;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CSharp;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using Microsoft.CodeAnalysis.Diagnostics;
+
+namespace CommunityToolkit.Maui.MediaElement.Analyzers;
+
+[DiagnosticAnalyzer(LanguageNames.CSharp)]
+public class UseCommunityToolkitMediaElementInitializationAnalyzer : DiagnosticAnalyzer
+{
+	public const string DiagnosticId = "MCTME001";
```

Comment:
Tests?

Reply:
Looks like you opened an issue for that. Outside of the scope of this PR.

Reply:
The issue is for another analyzer (yes, I know they are very similar).